pkg/apis/planetscale/v2/etcdlockserver_types.go

```diff
@@ -104,6 +104,10 @@ type EtcdLockserverTemplate struct {
 	// should be mounted.
 	ExtraVolumes []corev1.Volume `json:"extraVolumes,omitempty"`
 
+	// InitContainers can optionally be used to supply extra init containers
```

Comment:
Nit: Can you order these new fields after all the `Extra*` fields, so those similarly-named ones remain grouped?

Reply:
Sure, will do.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/19/commits/6f12c1b846c16e4b904f8d7d5f05fd5257bd5892.